packages/valory/skills/mech_interact_abci/models.py

```diff
@@ -40,6 +40,29 @@
 Requests = BaseRequests
 BenchmarkTool = BaseBenchmarkTool
 
+CHAIN_TO_NVM_CONFIG = {
+    "gnosis": {
+        "plan_fee_nvm": 10000000000000000,
+        "plan_price_mech": 990000000000000000,
+        "subscription_credits": 1000000,
+        "subscription_nft_address": "0x1b5DeaD7309b56ca7663b3301A503e077Be18cba",
+        "nft_sales_address": "0x72201948087aE83f8Eac22cf7A9f2139e4cFA829",
+        "subscription_token_address": "0x0000000000000000000000000000000000000000",
+        "subscription_provider_address": "0x4a2f40E14309c20c0C3803c3CcCd5E9B5F2D4eCA",
+        "plan_did": "did:nv:b0b28402e5a7229804579d4ac55b98a1dd94660d7a7eb4add78e5ca856f2aab7",
+    },
+    "base": {
+        "plan_fee_nvm": 10000,
+        "plan_price_mech": 990000,
+        "subscription_credits": 1000000,
+        "subscription_nft_address": "0xd5318d1A17819F65771B6c9277534C08Dd765498",
+        "nft_sales_address": "0x468dC6d758129c4563005B49aC58DfF2e6f7F08e",
+        "subscription_token_address": "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",
+        "subscription_provider_address": "0x5050c577583D25Ff9C9492A39e8D1B94028ffA55",
+        "plan_did": "did:nv:6f74c18fae7e5c3589b99d7cd0ba317593f00dee53c81a2ba4ac2244232f99da",
+    },
+}
+
```

Comment:
This should be configurable. Also, please create a `NVMConfig` `dataclass` and set it up in the `__init__` of the `MechParams`, to safely access these values later via the object.

Reply:
All the configs are static and so there is no point of keeping them configurable. All of these are tight knitted and if one of them have a mismatch, will cause more issues